desktop/core/src/desktop/js/apps/tableBrowser/metastoreTable.js

```diff
@@ -28,6 +28,9 @@ import I18n from 'utils/i18n';
 
 let contextPopoverTimeout = -1;
 
+export const TABLE_SOURCE_TYPE_HIVE = 'hive';
```

Comment:
@Harshg999 
There is also a `DIALECT` const in `desktop/core/src/desktop/js/apps/editor/snippet.js?` but that one doesn't contain `sparksql`. Is the dialect not the same as the source in this case or did we miss something here?

Reply:
Dialect can sometimes be flaky and have spaces in the value. For example, `spark sql`.
I feel we cannot rely on this param for condition checks and `type` can be a reliable one.

Reply:
Talked to Johan about this. He wanted us to use the connector dialect instead of the source type, I think to be more future proof. I will update the PR accordingly